### PR TITLE
Debain 10 buster support

### PR DESCRIPTION
**Pull request by agdod**
hi 
myself  ( @agdod ) and  @Johuotar - have added support for debian10 along with php7.3, made a few adjustments - service converted to systemctl , jailkit and antivirus to get it running with Debian 10. There might still be some issues we haven't come across.

### CODE REVIEW

**Comment by gOOvER** [comment]
It would be better, when Version is not hardcoded for phpMyAdmin. Maybe u can add a Variable or read Version from Website ;)

File: distros/debian10/install_phpmyadmin.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#---------------------------------------------------------------------
+# Function: InstallphpMyAdmin
+#    Install and configure phpMyAdmin
+#---------------------------------------------------------------------
+InstallphpmyAdmin() {
+    echo -n "Install myPhpAdmin...."
```
Comment (gOOvER) [inline distros/debian10/install_phpmyadmin.sh]: Installing phpMyAdmin...   

would be better

File: distros/debian10/install_phpmyadmin.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    cd /tmp
+    wget https://files.phpmyadmin.net/phpMyAdmin/4.9.0.1/phpMyAdmin-4.9.0.1-all-languages.tar.gz
+    tar xfz phpMyAdmin-4.9.0.1-all-languages.tar.gz
+    mv phpMyAdmin-4.9.0.1-all-languages/* /usr/share/phpmyadmin/
+    rm phpMyAdmin-4.9.0.1-all-languages.tar.gz
+    rm -rf phpMyAdmin-4.9.0.1-all-languages
+    cp /usr/share/phpmyadmin/config.sample.inc.php  /usr/share/phpmyadmin/config.inc.php
+    sed -i "$cfg['blowfish_secret'] = 'bD3e6wva9fnd93jVsb7SDgeiBCd452Dh'; /* YOU MUST FILL IN THIS FOR COOKIE AUTH! */" /usr/share/phpmyadmin/config.inc.php
+    sed -i "$cfg['TempDir'] = '/var/lib/phpmyadmin/tmp';" /usr/share/phpmyadmin/config.inc.php
+    echo -n " create apache configuration file for myPHPadmin"
```
Comment (gOOvER) [inline distros/debian10/install_phpmyadmin.sh]: Better would be:

"Creating Apache config file for PhpMyAdmin"

File: distros/debian10/install_phpmyadmin.sh (hunk 1 above)
Comment (agdod) [reply distros/debian10/install_phpmyadmin.sh]: done

Comment (agdod) [reply distros/debian10/install_phpmyadmin.sh]: actually i removed as its in the webserver script that calls the installation of phpmyadmin

File: distros/debian10/install_phpmyadmin.sh (hunk 2 above)
Comment (agdod) [reply distros/debian10/install_phpmyadmin.sh]: done.

**Comment by agdod** [comment]
all issues appear to be resolved. install of phpmyadmin is now successful

**Comment by servisys** [comment]
Hi guys, thank you very much for help!

I'll marge the commit, as usual, report eventually problem :)

**Comment by agdod** [comment]
no problem nice little learning project :) small request could you add us as contributors to the end of the readme file please? :)

**Comment by schliflo** [comment]
@agdod thank you for implementing Debian 10 support. Imho you should update the supported software table in the readme to contain buster: https://github.com/servisys/ispconfig_setup/blob/master/README.md#supported-software-and-linux-distributions

**Comment by agdod** [comment]
thanks for spotting the debain9- 10 changes that somehow got missed?!
honestly i didnt add it to the table as ive not been able to test all formats for instance not tested ngnix. focused on apache as that was what was requested for our server deployment. ill look into it.

**Pull request merged**